include/geode/mesh/helpers/convert_surface_mesh.h

```diff
@@ -53,4 +56,41 @@ namespace geode
     template < index_t dimension >
     void triangulate_surface_mesh( const SurfaceMesh< dimension >& surface,
         SurfaceMeshBuilder< dimension >& builder );
+
+    std::unique_ptr< SurfaceMesh3D >
+        opengeode_mesh_api convert_surface_mesh2d_into_3d(
+            const SurfaceMesh2D& surface2d, index_t axis_to_add );
+
+    std::unique_ptr< SurfaceMesh2D >
+        opengeode_mesh_api convert_surface_mesh3d_into_2d(
+            const SurfaceMesh3D& surface3d, index_t axis_to_remove );
+
+    std::unique_ptr< PolygonalSurface3D >
+        opengeode_mesh_api convert_polygonal_surface2d_into_3d(
+            const PolygonalSurface2D& surface2d, index_t axis_to_add );
+
+    std::unique_ptr< PolygonalSurface2D >
+        opengeode_mesh_api convert_polygonal_surface3d_into_2d(
+            const PolygonalSurface3D& surface3d, index_t axis_to_remove );
+
+    std::unique_ptr< TriangulatedSurface3D >
+        opengeode_mesh_api convert_triangulated_surface2d_into_3d(
+            const TriangulatedSurface2D& surface2d, index_t axis_to_add );
+
+    std::unique_ptr< TriangulatedSurface2D >
+        opengeode_mesh_api convert_triangulated_surface3d_into_2d(
+            const TriangulatedSurface3D& surface3d, index_t axis_to_remove );
+
+    template < index_t dimension >
+    std::unique_ptr< SurfaceMesh< dimension > > merge_surface_mesh(
+        const SurfaceMesh< dimension >& surface );
+
+    std::unique_ptr< SurfaceMesh3D > opengeode_mesh_api
+        merge_closed_surface_mesh( const SurfaceMesh3D& surface );
```

Comment:
maybe add a comment to explain contract of these two functions. BTW is `merge_closed_surface_mesh` useful / safe?